src/olympia/users/urls.py

```diff
@@ -41,19 +45,22 @@
         RedirectView.as_view(pattern_name='users.login', permanent=True),
         name='users.register'),
     url('^migrate', views.migrate, name='users.migrate'),
-    url(r'^pwreset/?$', anonymous_csrf(auth_views.password_reset),
+    url(r'^pwreset/?$',
+        migration_on(anonymous_csrf(auth_views.password_reset)),
```

Comment:
I thought that might be the case so I looked up [`waffle_switch`](https://github.com/jsocol/django-waffle/blob/5da47bbc1cffdbf84429ba046b059814c2e8e171/waffle/decorators.py#L34) and it looks like it doesn't do anything until the function is called. I added a test to show that it can flip without reloading urls.py.

I don't think it will hit the database until the view is accessed.